### PR TITLE
Add UG documentation of flip and test commands

### LINKED ISSUE #13: Update user guide documentation: Flip and Test commands
see https://nus-cs2103-ay2122s1.github.io/website/schedule/week7/project.html#3-update-the-ug

### PR DESCRIPTION
**Pull request by aikenwx**
### Description

Update user guide documentation for flip and test commands to be added in v1.2

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code quality improvements
- [x] Update documentation

### Checklist

- [x] I have updated the User Guide
- [x] I have included a preview deployment link to the updated project documentation: https://tp-rh03bdva3-aikenwx.vercel.app/UserGuide.html

closes #13

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```
Comment (chownces) [inline docs/UserGuide.md:24]: We can stick to using all 1s. Markdown automatically handles the numbering for us. This allows us to perform minimal changes if updates are needed in the future

File: docs/UserGuide.md [hunk 2]
```diff
@@ -33,8 +33,13 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
    * **`exit`** : Exits the app.
+   
+   * **`test`**`17`**`hello`**: Checks flashcard 17's translation against the word **`hello`** and then shows 
+     whether it is correct.
+   
+   * **`flip`**`2`: Toggles the 2nd flashcard to hide or show the correct translation.
 
-1. Refer to the [Features](#features) below for details of each command.
+6. Refer to the [Features](#features) below for details of each command.
```
Comment (chownces) [inline docs/UserGuide.md:42]: see comment above

File: docs/UserGuide.md [hunk 3]
```diff
@@ -154,6 +159,23 @@ Exits the program.
 
 Format: `exit`
 
+### Flipping a flashcard : `flip`
+
+* Flips the flashcard at the specified `INDEX` to either show or hide the translation to the displayed word or phrase.
+* Similar to delete, the index refers to the index number shown in the displayed flashcard list.
+* Similar to delete, the index **must be a positive integer** 1, 2, 3, …​
```
Comment (chownces) [inline docs/UserGuide.md:166]: Can we follow the format of the rest of the documentation? For example

```
### Deleting a person : `delete`
Deletes the specified person from the address book.
Format: `delete INDEX`
* Deletes the person at the specified `INDEX`.
* The index refers to the index number shown in the displayed person list.
* The index **must be a positive integer** 1, 2, 3, …​
```

I have deployed this commit `6de3fa1` here for reference: https://a8d12ad0.tp-b32.pages.dev/

File: docs/UserGuide.md [hunk 4]
```diff
@@ -154,6 +159,23 @@ Exits the program.
...
+Format: `flip INDEX`
+
+### Testing with a flashcard : `test`
+
+* Checks the translation of the flashcard at the specified `INDEX` with the given `WORD`.
+* The app will then show user the correct translation and tell the user whether he got it right.
+* We can only against with a flashcard that is flipped down (i.e. the translation is hidden).
+* The `WORD` is not case-sensitive if given in ASCII characters (e.g. "HeLLo" matches "hello"). 
+
+Format: `test INDEX WORD`
```
Comment (chownces) [inline docs/UserGuide.md:177]: see comment above on format

**Review by chownces — CHANGES_REQUESTED** [review]

**Comment by chownces** [comment]
Command summary at the bottom of UserGuide needs to be updated too

File: docs/UserGuide.md [hunk 5]
```diff
@@ -161,20 +164,30 @@ Format: `exit`
...
 
-Format: `flip INDEX`
+Examples:
+* `flip 3` followed by `flip 3` shows and then hides the English translation for the 3rd flashcard on display.
 
 ### Testing with a flashcard : `test`
 
+Checks whether the English translation of a flashcard matches a specified English word.
+
+Format: `test INDEX WORD`
```
Comment (kwokieee) [inline docs/UserGuide.md:182]:
```suggestion
Format: `test INDEX ENGLISH_PHRASE`
```

Let's standardize across all commands and use `ENGLISH_PHRASE` for now

**Review by kwokieee — CHANGES_REQUESTED** [review]

**Review by kwokieee — APPROVED** [review]

**Review by chownces — APPROVED** [review]

**Pull request merged**